_data/tutorials.yaml

```diff
@@ -806,3 +806,14 @@ multiple-event-types:
     kafka: enabled
     confluent: enabled
 
+
+fleet-management:
+  title: "Optimize Fleet Management"
+  canonical: confluent
+  meta-description: "Optimize Fleet Management"
+  slug: "/fleet-management"
+  question: "Optimize Fleet Management"
+  introduction: "More and more, fleet management relies on knowing real-time information on vehicle availability, their locations, and integrating that with data from vehicle telematics. This enables businesses to improve operational efficiency by optimizing travel routes, lowering fuel consumption, and automating service schedules. This recipe combines fleet locations with individual vehicle information, so organizations can have a real-time consolidated view of their entire fleet. <center><img src='/prototype-recipe/assets/img/fleet.png'></center>"
```

Comment:
@colinhicks I was wondering if you had guidance here.

```
<img src='/prototype-recipe/assets/img/fleet.png'>
```

This uses a static path `/prototype-recipe` because that's the name of the path for this PR.  Is there any way to make this dynamic and use something similar to `| relative_url`?

Reply:
I think `src="{{ "/assets/img/fleet.png" | relative_url }}"` should work. The `relative_url` logic uses the base URL, which is set to the branch name by the CI build: https://github.com/confluentinc/kafka-tutorials/blob/e354bca72485ae63ce56e582cca9de4910fb7ac9/.semaphore/semaphore.yml#L58

Reply:
@colinhicks user error.  I was building incorrectly!  Now fixed

Reply:
@ybyzek, sorry, I misunderstood the full context for this change. My suggestion is to 1) treat the intro image path as data and 2) move the relevant html into `_layouts/recipe.html`. 

1)
```suggestion
  introduction: "More and more, fleet management relies on knowing real-time information on vehicle availability, their locations, and integrating that with data from vehicle telematics. This enables businesses to improve operational efficiency by optimizing travel routes, lowering fuel consumption, and automating service schedules. This recipe combines fleet locations with individual vehicle information, so organizations can have a real-time consolidated view of their entire fleet."
  introduction_media: "assets/img/fleet.png"
```

2) in `_layouts/recipe.html`
```diff
        <div class="example">
         <p class="text">{{ site.data.tutorials[page.static_data].introduction }}</p>
+        <img src="{{ site.data.tutorials[page.static_data].introduction_media | relative_url }}" />
        </div>
```
(You could add some other markup around the img tag to center it and/or add other styles via the stylesheet.)

Reply:
@colinhicks brilliant!  This worked like a charm, thank you so much for explaining and offering this solution.  Updated in latest commit.